data/meson.build

```diff
@@ -14,9 +14,15 @@ desktop_file = i18n.merge_file(
   install_dir: join_paths(datadir, 'applications')
 )
 
-desktop_utils = find_program('desktop-file-validate', required: false)
-if desktop_utils.found()
-  test('Validate desktop file', desktop_utils, args: [desktop_file])
+desktop_file_validate = find_program('desktop-file-validate', required: false)
+if desktop_file_validate.found()
+  test(
+    'validate-desktop',
```

Comment:
In my understanding this is the display name, this should be the one it was before imo

Reply:
I think this should work. The previous tests for the desktop file and the schema were not working at all, they actually don't in the default GNOME Builder Python template. 

The solution I've used here, I've adapted from GNOME Sound Recorder, see [there meson.build file in data](https://gitlab.gnome.org/World/vocalis/-/blob/master/data/meson.build?ref_type=heads).

Reply:
I meant just the display name (what you see in GNOME Builder in Unit tests). Changing the display name to something more out of the english language would be the ideal here. A quick check shows no breakage there. See l. 59

Reply:
Ah, right. Yeah you are absolutely right, I agree here. The issue was likely simply that I was looking elsewhere how to solve the issue I was facing, and went a bit too far when adopting their solution.